cmake/Modules/LocalJpeg.cmake

```diff
@@ -53,6 +62,14 @@ else()
     )
     add_dependencies(JPEG::JPEG libjpeg)
     set(JPEG_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/libjpeg/src/libjpeg)
+    if(IS_MULTI_CONFIG)
+        set_target_properties(JPEG::JPEG PROPERTIES IMPORTED_LOCATION_DEBUG "${LIB_FILENAME_DEBUG}" AVIF_LOCAL ON)
+        set_target_properties(JPEG::JPEG PROPERTIES IMPORTED_LOCATION_MINSIZEREL "${LIB_FILENAME_MINSIZEREL}" AVIF_LOCAL ON)
+        set_target_properties(JPEG::JPEG PROPERTIES IMPORTED_LOCATION_RELEASE "${LIB_FILENAME_RELEASE}" AVIF_LOCAL ON)
+        set_target_properties(
+            JPEG::JPEG PROPERTIES IMPORTED_LOCATION_RELWITHDEBINFO "${LIB_FILENAME_RELWITHDEBINFO}" AVIF_LOCAL ON
+        )
+    endif()
```

Comment:
Vincent: Ideally we would change line 75 to use a generator expression that contains `$<CONFIG>` as the value of the [`IMPORTED_LOCATION`](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_LOCATION.html) property. Unfortunately the value of `IMPORTED_LOCATION` cannot be a generator expression. So using `IMPORTED_LOCATION_<CONFIG>` is the best solution I came up with.

Reply:
I could not find a better solution either. I am not sure a for loop would be cleaner.